### PR TITLE
add Haswell-noTSX and Broadwell-noTSX

### PR DESCRIPTION
**Pull request by XueqiangWei**
Signed-off-by: XueQiangWei <xuwei@redhat.com>

### CODE REVIEW

**Comment by XueqiangWei** [comment]
ID: 1257821

**Comment by xutian** [comment]
@XueqiangWei  pep8 check failed, please fix and update your branch.

thanks,
Xu

**Comment by XueqiangWei** [comment]
@xutian got it, thanks

Xueqiang Wei

File: virttest/utils_misc.py [hunk 1]
```diff
@@ -2091,6 +2092,8 @@ def _make_up_pattern(flags):
                    "Opteron_G3": "cx16,sse4a",
                    "Opteron_G2": "cx16",
                    "Opteron_G1": "",
+                   "Broadwell-noTSX": "adx,rdseed,3dnowprefetch",
+                   "Haswell-noTSX": "fma,avx2,movbe",
                    "Broadwell": "adx,rdseed,3dnowprefetch",
```
Comment (suqinhuang) [inline virttest/utils_misc.py]: what's the different between  Broadwell-noTSX and Broadwell

Comment (XueqiangWei) [inline virttest/utils_misc.py]: Broadwell-noTSX: disables the HLE and RTM features on Broadwell CPU

Comment (suqinhuang) [inline virttest/utils_misc.py]: if so, hle and rtm should be suppored in Broadwell

**Comment by suqinhuang** [comment]
Acked-by: Suqin Huang

**Comment by xutian** [comment]
looks good to me, merged thanks @XueqiangWei  and @suqinhuang

**Pull request merged**